adapters/aidem/aidem.go

```diff
@@ -56,7 +54,7 @@ func (a *adapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest
 		}}
 	}
 
-	bidResponse := adapters.NewBidderResponse()
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(bidResp.SeatBid[0].Bid))
```

Comment:
IMO, you would want to check if there are actually SeatBids being returned from your bidder server. A check like below should work - 
```
if len(bidResp.SeatBid) == 0 {
		return nil, []error{&errortypes.BadServerResponse{
			Message: "Empty SeatBid array",
		}}
	}
```

Reply:
If no bids are availabel a 204 response will come from the server  
`if adapters.IsResponseStatusCodeNoContent(response) {
   return nil, nil
}`

Reply:
Please consider this case - 
If bidder returns a response with status as 200 but no SeatBids then line 57 will cause a panic. I know this is going to be a rare case but it is quite possible. So, requesting you to please add a check.

Reply:
@gargcreation1992 Implemented as requested.